templates/navigation.html

```diff
@@ -10,9 +10,9 @@
                 {% if current_page == "index" %}
                     <li class="uk-active"><a href="/"><i class="uk-icon fas fa-home"></i> &nbsp;Home</a></li>
                 {% else %}
-                    <li><a href="/"><i class="uk-icon fas fa-home"></i> &nbsp;Home</a></li>
+                    <li><a href="https://pythondiscord.com/"><i class="uk-icon fas fa-home"></i> &nbsp;Home</a></li>
```

Comment:
Why?

Reply:
I accidentally left this in, but it was added because if we use this navbar on any of the subdomains, they will have unexpected behavior.

Reply:
I'm not sure why we'd use this navbar on other domains